server/utils/nunjucksSetup.ts

```diff
@@ -5,14 +5,16 @@ import express from 'express'
 import { initialiseName } from './utils'
 import config from '../config'
 
-const production = process.env.NODE_ENV === 'production'
-
 export default function nunjucksSetup(app: express.Express, path: pathModule.PlatformPath): void {
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
   app.locals.applicationName = 'Launchpad'
 
+  app.locals.environment = process.env.NODE_ENV
```

Comment:
Can't this be pulled from the config file that's already imported? https://github.com/ministryofjustice/hmpps-launchpad-home-ui/blob/b13d52ccb2ba471dddd55b4960141a39dfce4efd/server/config.ts#L33